### PR TITLE
Updated for OpenProject 11.1

### PR DESCRIPTION
**Pull request by b12f**
## :star2: What does this PR do?

OpenProject 11.1 updated the work package detail page, removing the description heading that was previously used to show the toggl button. This PR moves the toggle button to the toolbar at the top of the work package page, keeping compatability with OP < 11.1, while also gaining compatability with future versions. I also think this generally is a better place for the button, as it is closer to where OP's built-in timetracking features are also presented.

## :bug: Recommendations for testing

All changes should be tested across Chrome and Firefox.

## :memo: Links to relevant issues or information

https://github.com/opf/openproject/releases/tag/v11.1.0

### CODE REVIEW

**Comment by rylek90** [comment]
Hey @b12f, thank you for contribution and sorry for keep you waiting for such a long time :scream: 

I have troubles with finding button in OpenProject, can you maybe attach some screenshots to help me? :pray:

**Comment by b12f** [comment]
Hi @rylek90, 

I did the following:
1. Building my changes locally,
2. loading the extension locally (in FF),
3. logging into toggle,
4. enabling the OpenProject integration,
5. opening the following page: https://community.openproject.com/projects/openproject/work_packages/34852/activity?query_id=2453
6. 
I get the button in the top right, as part of the toolbar items:

**Review by rylek90 — APPROVED** [review]
I am blind :joy: 
Thank you for quick response :raised_hands: 

It's good to go for me, I can see it in my own created account as well now.

One more question (totally not request for change): maybe you find it useful to add more `margin-right`? :thinking: 
Looks like elements are not evenly spaced now
image
image

**Comment by b12f** [comment]
Done! Thanks for the response by the way :)

**Comment by rylek90** [comment]
Woohoo! We'll merge it right a way as soon as it'll get second approval. And **thank you** for contribution and patience :bow:

**Comment by wielinde** [comment]
Friendly reminder ping for @nunofmn to please review this tiny PR.

**Review by nunofmn — APPROVED** [review]
LGTM! 👍 🚀

**Comment by wielinde** [comment]
Thank you @b12f, @rylek90, and @nunofmn!!

**Pull request merged**
